src/Forex/ForexData.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+using Azure.Migrate.Export.Common;
+using Azure.Migrate.Export.HttpRequestHelper;
+using Azure.Migrate.Export.Models;
+
+namespace Azure.Migrate.Export.Forex
+{
+    public class ForexData
+    {
+        private static Dictionary<string, double> ExchangeRatesUSD = null;
+
+        public static double GetExchangeRate(UserInput userInputObj)

```

Comment:
Here we should just pass currency, not the whole userInputObj.

Reply:
There are three reasons for passing UserInput parameter here:
1. Firstly is the CurrencySymbol (USD, INR, etc)
2. The logger object is also a part of this parameter, hence to record warning / errors during API calls require it for updating the logs.
3. If everything fails, and we are not able to obtain any exchange rate -> we need to reset UserInputObject.Currency to USD.

Reply:
Understood, Implemented ForexData with singleton parameter.
Thanks.